naming/src/main/java/com/alibaba/nacos/naming/core/ServiceOperatorV2Impl.java

```diff
@@ -91,8 +91,13 @@ public void delete(String namespaceId, String serviceName) throws NacosException
     
     @Override
     public ObjectNode queryService(String namespaceId, String serviceName) throws NacosException {
-        ObjectNode result = JacksonUtils.createEmptyJsonNode();
         Service service = getServiceFromGroupedServiceName(namespaceId, serviceName, true);
+        boolean serviceExist = ServiceManager.getInstance().containSingleton(service);
+        if (!serviceExist) {
```

Comment:
merge `serviceExist`, because `serviceExist` will not be used after this logic.

Reply:
OK, I will merge it.

Reply:
> merge `serviceExist`, because `serviceExist` will not be used after this logic.